src/main/java/com/algorand/algosdk/v2/client/model/DryrunApp.java

```diff
@@ -0,0 +1,35 @@
+package com.algorand.algosdk.v2.client.model;
```

Comment:
this file should not be created, DryrunApp is gone and has no references

Reply:
Thanks for catching this.

Reply:
Looks like we're good now. I didn't see anything while grepping around for "DryRun" and "DryrunApp". Hopefully these are the only unused files that snuck in.